docs/HowToFAQ/FAQSetToSeq.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Is there a nice way to turn a set into a seq?
+---
+
+## Question
+
+Is there a nice way to turn a set into a seq?
+
+## Answer
+
+There is as yet no built-in simple function but there are various idioms that can accomplish this task.
+
+Within a method (where you can use a while loop), try `var set acc = {}; while s != {} { var x :| x in s; acc := acc + [x] }`
```

Comment:
This example seems to be missing an update of `s` in the loop.

Reply:
And more problems than that -- I should have put the snippet in VSCode instead of the C&P from Slack!